### ISSUE TITLE
QC report- Add check for clincial collection data received but samples not entered in Dashboard

### ISSUE THREAD

**Issue by erincschwartz** [Analyticsphere/metricsReportsRequests#185]
Please add 2 new rules (1 for blood, 1 for urine) to catch cases where clincial back end data are sent but samples are not entered into the Dashboard: 

1) If BioClin_SiteBloodCollBL_v1r0 is yes, then BioFin_BaseBloodCol_v1r0 should be yes 
2) If BioClin_SiteUrineCollBL_v1r0 is yes, then BioFin_BaseUrineCol_v1r0 should be yes

**Comment by KELSEYDOWLING7** [comment]
Got it! There are 17 errors for the first and 56 errors for the second. Is that about what you were expecting?

**Comment by KELSEYDOWLING7** [comment]
CSV files for these participants have been added here: https://nih.app.box.com/folder/249010743726

**Comment by KELSEYDOWLING7** [comment]
If approved by Wednesday 12/4, this can be updated in the automation for Monday's report

**Comment by erincschwartz** [comment]
Can you please add the following variables to the spreadsheet for Github Issue 185 Blood Errors ASAP? 


BioClin_SntBloodAccIDBL_v1r0	
BioClin_SntUrineAccIDBL_v1r0	
BioClin_PolyBloodIDBL_v1r0	
BioClin_PolyUrineIDBL_v1r0 
BioClin_ClinBloodTmBL_v1r0

**Comment by KELSEYDOWLING7** [comment]
Here you go https://nih.app.box.com/file/1717886905886

**Comment by KELSEYDOWLING7** [comment]
Actually I need to go back and add SiteBioFin_BaseBloodCol_v1r0BioClin_SiteBloodCollBL_v1r0Finalized_Date but these are likely (mostly) the same participants from before

**Comment by KELSEYDOWLING7** [comment]
Now all the variables are there: https://nih.app.box.com/file/1717901721466

**Comment by KELSEYDOWLING7** [comment]
@erincschwartz  Is this rule good as is to add to the report? Or do we need to add any other exclusions or logic to the rule?

**Comment by KELSEYDOWLING7** [comment]
@erincschwartz  Following up on this issue. The blood rule has only 6 issues now, and the urine rule 39

**Comment by erincschwartz** [comment]
@KELSEYDOWLING7 Picking this issue back up, is it easy to run a quick report again to see the status of these new rules? Then I will make a timely decision. Sorry to leave this hanging. I do believe we will need these rules added.

**Comment by KELSEYDOWLING7** [comment]
Yes, those have been uploaded here: https://nih.app.box.com/file/1824535127916

**Comment by KELSEYDOWLING7** [comment]
@erincschwartz Was anything decided on these new rules?

**Comment by KELSEYDOWLING7** [comment]
Confirmed this is the same as rules 2a and 2b. Closing this issue

**Comment by KELSEYDOWLING7** [comment]
Discussed over teams, concluded that blood issues already being included in 29.b and urine issues already being included in 30b
